charts/osm/templates/osm-bootstrap-deployment.yaml

```diff
@@ -74,8 +74,6 @@ spec:
               containerPort: 9443
             - name: "metrics"
               containerPort: 9091
-            - name: "health"
```

Comment:
I get cutting down on unnecessary ports, but using the metrics port for healthchecks feels weird to me

Reply:
Fair enough; yeah meta or admin makes sense to me, but I won't block this PR on it